lib/r10k/cli/puppetfile.rb

```diff
@@ -46,7 +46,7 @@ def self.command
           usage 'check'
           summary 'Try and load the Puppetfile to verify the syntax is correct.'
 
-          required nil, :puppetfile, 'Path to Puppetfile (can also be set with PUPPETFILE environment variable)'
```

Comment:
D'oh. You fooled me just like the 2.5.x branch did. Those helpful parenthetical phrases already don't exist in the 2.5.x branch.